pam/integration-tests/vhs-helpers_test.go

```diff
@@ -187,7 +189,24 @@ func (td tapeData) ExpectedOutput(t *testing.T, outputDir string) string {
 		}
 	}
 
-	return permissionstestutils.IdempotentPermissionError(got)
+	got = permissionstestutils.IdempotentPermissionError(got)
+
+	// Drop all the empty lines before each page separator, to remove the clutter.
+	got = vhsEmptyLinesRegex.ReplaceAllString(got, "$4")
+
+	// Save the sanitized result on cleanup
```

Comment:
There are some golden files which were not sanitized, I suspect that's because those are leftovers from old tests:

```
pam/integration-tests/testdata/TestCLIAuthenticate/golden/authenticate_user_and_reset_password
pam/integration-tests/testdata/TestCLIChangeAuthTok/golden/change_password_successfully
pam/integration-tests/testdata/TestCLIChangeAuthTok/golden/deny_authentication_if_user_does_not_exist
pam/integration-tests/testdata/TestNativeAuthenticate/golden/authenticate_user_and_reset_password
```

Maybe you want to run:
```
git rm "pam/integration-tests/testdata/*golden*"
TESTS_UPDATE_GOLDEN=1 go test ./pam/integration-tests/...
```

Reply:
Done, it's some time I'm not doing this... Ideally we should add a test that checks that there are no unexpected golden files too. It's a bit annoying though.

Reply:
I've a strategy btw, I'll try to handle it soon

Reply:
> Ideally we should add a test that checks that there are no unexpected golden files too.

Yes that would be useful

> I've a strategy btw, I'll try to handle it soon

Thanks!